tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 skipsdist=True
-envlist = lint, unit, func
+envlist = lint
```

Comment:
We are using func in each other project and I do not understand why you think this:
"avoid running func tests by default (one may not wish to run func tests on their workstation, so it may be dangerous to run it by default)"

If you do not know what tox is running, you should never run it.

Reply:
It feels like a footgun to default to running something that directly makes changes to the host machine.  If this was running in a container, VM, temporary juju model, etc. I'd agree with you.

Reply:
I agree, we should have safe defaults and functests generally aren't